src/plugins/vis_type_tagcloud/public/components/collections.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { TagCloudVisParams } from '../types';
+
+interface Scales {
+  text: string;
+  value: TagCloudVisParams['scale'];
+}
+
+interface Orientation {
+  text: string;
+  value: TagCloudVisParams['orientation'];
+}
+
+interface Collections {
+  scales: Scales[];
+  orientations: Orientation[];
+}
+
+export const collections: Collections = {
+  scales: [
+    {
+      text: i18n.translate('visTypeTagCloud.vis.editorConfig.scales.linearText', {
+        defaultMessage: 'Linear',
+      }),
+      value: 'linear',
+    },
+    {
+      text: i18n.translate('visTypeTagCloud.vis.editorConfig.scales.logText', {
```

Comment:
nit: @stratoula  are you ok to keep current id's visTypeTagCloud.vis.`editorConfig`.scales.logText' . I think it would be semantically correct to rename `vis.editorConfig` -> `collections`

Reply:
Agree @alexwizp

Reply:
done